### PR TITLE
Add capabilities for performance tuning

### PR DESCRIPTION
**Pull request by Savidude**

### CODE REVIEW

File: modules/apim_common/files/system/etc/security/limits.conf [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+#root            hard    core            100000
+#*               hard    rss             10000
+#@student        hard    nproc           20
+#@faculty        soft    nproc           20
+#@faculty        hard    nproc           50
+#ftp             hard    nproc           0
+#ftp             -       chroot          /ftp
+#@student        -       maxlogins       4
+
+# End of file
```
Comment (VimukthiPerera) [inline modules/apim_common/files/system/etc/security/limits.conf]: Please add an empty line to the EOF.

File: modules/apim_common/files/system/etc/sysctl.conf [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+#
+# Do not accept IP source route packets (we are not a router)
+#net.ipv4.conf.all.accept_source_route = 0
+#net.ipv6.conf.all.accept_source_route = 0
+#
+# Log Martian Packets
+#net.ipv4.conf.all.log_martians = 1
+#
+
+###################################################################
```
Comment (VimukthiPerera) [inline modules/apim_common/files/system/etc/sysctl.conf]: Add empty line to the EOF.

Comment (Savidude) [reply modules/apim_common/files/system/etc/sysctl.conf]: Fixed https://github.com/wso2/puppet-apim/pull/132/commits/748e35c927d951205b8e13a7ea77d0e37f0479a5

File: modules/apim_common/files/system/etc/security/limits.conf (hunk 1 above)
Comment (Savidude) [reply modules/apim_common/files/system/etc/security/limits.conf]: Fixed https://github.com/wso2/puppet-apim/pull/132/commits/748e35c927d951205b8e13a7ea77d0e37f0479a5

**Review by VimukthiPerera — APPROVED** [review]

**Pull request merged**